target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490-micron.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritz5490.dts"
```

Comment:
I don't remember if including dts is a good solution.

Reply:
Its probably not. I have looked again at how its done with the 3370 and maybe this is the way to do. Given that some port configuration differ and your comment about 5490, its probably better to add a separate include for each device and then a micron and non-micron NAND dts. Thats more text, but likely cleaner.

Reply:
I know some dts files in mainline linux, that do this as well. See this one for example: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm/boot/dts/stm32mp157c-ev1.dts

What is against including a dts?

Reply:
A common dtsi should be used whenever devices share multiple identical nodes, however the name should be generic, e.g. `vr9_avm_fritz-x490-micron.dtsi` (the dash actually does not belong there, since that's not consistent with the model name, it just looked weird without it).

Even for the `micron` variants there probably should be a generic dtsi that includes the non-micron variant, to further reduce redundancy among multiple device variants.

Reply:
I have restructured the includes and removed all parts that need to be overwritten from the common include and created dtsi for each of the devices to be included in the micron and non-micron variants.